Content.Server/Anomaly/Components/AnomalySynchronizerComponent.cs

```diff
@@ -0,0 +1,42 @@
+using Content.Shared.Anomaly;
+using Content.Shared.Anomaly.Components;
+using Content.Shared.DeviceLinking;
+using Robust.Shared.Audio;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.Anomaly.Components;
+
+/// <summary>
+/// a device that allows you to translate anomaly activity into multitool signals.
+/// </summary>
+[RegisterComponent, Access(typeof(AnomalySynchronizerSystem))]
+public sealed partial class AnomalySynchronizerComponent : Component
+{
+    /// <summary>
+    /// The uid of the anomaly to which the synchronizer is connected.
+    /// </summary>
+    [DataField, ViewVariables(VVAccess.ReadWrite)]
+    public EntityUid? ConnectedAnomaly;
+
+
+    [DataField, ViewVariables(VVAccess.ReadOnly)]
+    public ProtoId<SourcePortPrototype> DecayingPort = "Decaying";
+
+    [DataField, ViewVariables(VVAccess.ReadOnly)]
+    public ProtoId<SourcePortPrototype> StabilizePort = "Stabilize";
+
+    [DataField, ViewVariables(VVAccess.ReadOnly)]
+    public ProtoId<SourcePortPrototype> GrowingPort = "Growing";
+
+    [DataField, ViewVariables(VVAccess.ReadOnly)]
+    public ProtoId<SourcePortPrototype> PulsePort = "Pulse";
+
+    [DataField, ViewVariables(VVAccess.ReadOnly)]
+    public ProtoId<SourcePortPrototype> SupercritPort = "Supercritical";
```

Comment:
You don't need to specify `ViewVariables(VVAccess.ReadOnly)` for any fields that also have a `DataField` attribute, those implicitly have VV read only.

Reply:
Done